spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -186,6 +186,10 @@ private StandardTableDefinition testPartitionedTableDefinition() {
     return bq.getTable(testDataset.toString(), testTable + "_partitioned").getDefinition();
   }
 
+  private StandardTableDefinition getClusteredTableDefinition() {
+    return bq.getTable(testDataset.toString(), testTable + "_clustered").getDefinition();
```

Comment:
Why is it needed?

Reply:
This is just a helper method, to reduce the amount of work being done in test and if we want to have more tests around clusteredTables later.

Reply:
Sorry, I'll clarify. Why do we need the `_clustered` suffix instead of using the generated random `testTable` name?

Reply:
Ah I misunderstood the previous comment. Fixed